spec/features/user_spec.rb

```diff
@@ -27,5 +26,21 @@
     expect(page).to have_content "Sorry, you aren't allowed to do that."
   end
 
+  scenario "should allow user who is an admin to export list of users" do
+    user.add_role :admin
+    sign_in user
+    visit "/users"
+    click_on 'Export to CSV'
+    header = page.response_headers['Content-Disposition']
+    expect(header).to match "attachment; filename=\"users-2019-08-03.csv\""
```

Comment:
we might want to interpolate the date here (and in the next spec too) 😸
if we run this spec tomorrow, it'll error out since it won't be `2019-08-03`
smth like this `....users-#{Date.today}.csv...`

Reply:
Oh snaps! Thanks for catching that! I'll change it. :-)

Reply:
Thanks @smurfik! `users-#{Date.today}.csv` worked perfectly! I appreciate it :-)